.github/workflows/ci.yaml

```diff
@@ -92,12 +92,11 @@ jobs:
       with:
         distribution: temurin
         java-version: 11
+        cache: maven
     - run: java -version
-    - uses: actions/cache@v3
-      id: mvn-cache
-      with:
-        path: ~/.m2/repository
-        key: ${{ runner.os }}-maven-unified-${{ steps.date.outputs.week_of_year }}
+    - name: Install Maven modules to local Maven repository
+      run: |
+        mvn install -B -ntp  -T 1C -DskipTests -Dclirr.skip -Dcheckstyle.skip
```

Comment:
Looks like a cache issue indeed
cc: @JoeWang1127 @blakeli0

Reply:
Otherwise we get an error about missing dependencies:
```
Error:  Failed to execute goal org.apache.maven.plugins:maven-enforcer-plugin:3.4.1:enforce (enforce) on project grpc-google-cloud-accessapproval-v1: 
Error:  Rule 3: org.codehaus.mojo.extraenforcer.dependencies.BanDuplicateClasses failed with message:
Error:  Could not find artifact com.google.api.grpc:proto-google-cloud-accessapproval-v1:jar:2.39.0-SNAPSHOT in google-maven-central-copy (https://maven-central.storage-download.googleapis.com/maven2)
Error:  -> [Help 1]
```